templates/fine-tune-llm/job_compute_configs/gcp.yaml

```diff
@@ -22,15 +22,9 @@ compute_config:
           a2-highgpu-8g-nvidia-a100-40gb-8: 8
   gcp_advanced_configurations_json:
     instance_properties:
-      labels:
-        as-feature-multi-zone: "true"
       disks:
         - boot: true
           auto_delete: true
           initialize_params:
             disk_size_gb: 250
-runtime_env:
```

Comment:
reason for removal?

Reply:
Those are auto-populated by llmforge and no longer needed. For instance, they are not needed on the aws file